lib/wiki_edit_mappings.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Class returns template mappings for WikiEdu and P&E Dashboards, respectively
+class WikiEditMappings
```

Comment:
This will let us get started on English Wikipedia for P&E Dashboard, but we'll want to be able to use different templates for each language / project. (For dashboard.wikiedu.org, it will remain enabled only on English Wikipedia.)